saleor/core/management/commands/dummy_command.py

```
@@ -0,0 +1,30 @@
+"""Clear the database preserving shop's configuration.
+
+This command clears the database from data such as orders, products or customer
+accounts. It doesn't remove shop's configuration, such as: staff accounts, service
+accounts, plugin configurations, site settings or navigation menus.
+"""
+
+from django.conf import settings
```

Ruff

`django.conf.settings` imported but unused

Remove unused import: `django.conf.settings`

(F401)